src-ts/tools/learn/free-code-camp/FreeCodeCamp.tsx

```diff
@@ -54,18 +85,35 @@ const FreeCodeCamp: FC<{}> = () => {
 
             {lesson && (
                 <Portal portalId='page-subheader-portal-el'>
-                    <div className={styles['iframe-wrap']}>
-                        <CollapsiblePane title='Course Outline'>
-                            <CourseOutline
-                                course={courseData}
-                                ready={courseDataReady}
-                                currentStep={`${searchParams.get('module')}/${searchParams.get('lesson')}`}
+                    <div className={styles['main-wrap']}>
+                        <div className={styles['course-outline-pane']}>
+                            <CollapsiblePane title='Course Outline'>
+                                <div className={styles['course-outline-wrap']}>
+                                    <div className={styles['course-outline-title']}>
+                                        {courseData?.title}
+                                    </div>
+                                    <CourseOutline
+                                        course={courseData}
+                                        ready={courseDataReady}
+                                        currentStep={`${searchParams.get('module')}/${searchParams.get('lesson')}`}
```

Comment:
let's either create an enum w/all possible values of the search params or reuse the moduleParam and lessonParam variables. 

we just want to avoid any magic strings like this.